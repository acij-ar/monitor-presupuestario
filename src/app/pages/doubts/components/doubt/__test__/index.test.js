const React = require('react');
const { shallow } = require('enzyme');

const Doubt = require('..');

describe('Doubt component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <Doubt title="Mocked doubt title">
        Mocked doubt content
      </Doubt>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
