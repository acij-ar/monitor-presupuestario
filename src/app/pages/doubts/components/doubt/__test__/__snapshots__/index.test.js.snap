// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Doubt component should match snapshot 1`] = `
<div
  className="doubt"
>
  <label>
    <input
      defaultChecked={false}
      name="doubts"
      type="radio"
    />
    <ContentSection
      title="Mocked doubt title"
    >
      Mocked doubt content
    </ContentSection>
  </label>
</div>
`;
